pages/about.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../css/mobile-screen.css">
+    <link rel="stylesheet" href="../css/about-mobile-screen.css">
+    <link rel="stylesheet" media="screen and (min-width: 768px)" href="../css/wide-screen.css">
+    <link rel="stylesheet" media="screen and (min-width: 768px)" href="../css/about-wide-screen.css">
+    <title>About</title>
+</head>
+
+<body>
+    <section class="header-background">
+        <header class="header">
+            <div class="social-media">
+                <a href="#" class="social-media-link">
+                    <img src="" alt="">
+                </a>
+                <a href="" class="social-media-link">
+                    <img src="" alt="">
+                </a>
+                <a href="" class="last" class="social-media-link"> English </a>
+            </div>
+            <nav class="nav-bar">
+                <a href="../index.html" class="logo"> <img src="../img/logo_muda.jpg" alt="" width="80px" height="70px"></a>
+                <ul class="nav-menu">
+                    <li class="nav-item nav-item-1">
+                        <a href="about.html" class="nav-link"> About</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link"> Program</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link"> Join</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link"> Sponsor</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link"> News</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link"> CC Campaign</a>
+                    </li>
+                    <div class="close-btn">
+                        <img src="../img/close.png" alt="Close burger menu">
+                    </div>
+                </ul>
+                <div class="burger-menu">
+                    <a href="#" class="burger-menu-btn">
+                        <img src="../img/menu.png" alt="Open menu" width="24px" height="17.5px">
+                    </a>
+                </div>
```

Comment:
- [ ] For About page of mobile screen size when I Clicked on hamburger menu nothing happens , it show an error on the console, Please try to fix it.
![Capture](https://user-images.githubusercontent.com/93598189/197209022-41b30da4-0ec7-4fab-a3e5-5dbbaddaf68d.PNG)